data-processing/common/commands/fetch_s2_data.py

```diff
@@ -59,13 +59,15 @@ def _mk_api_request(self, arxivId: ArxivId) -> requests.Response:
         headers = None
 
         if self._partner_api_token:
-            base_url = "partner.semanticscholar.org"
             headers = {"x-api-key": self._partner_api_token}
 
+        references_fields = ["authors", "title", "externalIds", "venue", "year"]
+        fields_query = "fields=references." + ",references.".join(references_fields)
+
         logger.info(f"Issuing request to S2 @ {base_url}")
 
         return requests.get(
-            f"https://{base_url}/v1/paper/arXiv:{versionless_id}",
+            f"https://{base_url}/graph/v1/paper/arXiv:{versionless_id}?{fields_query}",
```

Comment:
I'd suggest using the `params` parameter instead of building the query string yourself. That'll take care of escaping things that need to be escaped, and probably eliminates a few lines. There's probably nothing to escape here, but it's one less thing to think about.

Combined with the change above I think that'd produce something like this:

```python
return requests.get(
    "https://{base_url}/graph/v1/paper/arXiv:{versionless_id}",
    params={"fields": ",".join(fields) },
    headers=headers
)
```

Here's the relevant docs for the `requests` library:
https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls